### PR TITLE
Added a simple slots game command.

### PR DESCRIPTION
**Pull request by ChaoticNebula5**

### CODE REVIEW

**Review by tommodev-ctrl — COMMENTED** [review]
It could be improved but it's good. Also, for the color, you have to use the hex code of 1Bot's embeds.

**Comment by thatjar** [comment]
It's good... but as @Tommins-IT said, can you improve it a bit? You can say that you won if there's 3 sevens.

File: cogs/fun.py [hunk 1]
```diff
@@ -331,7 +331,19 @@ async def figlet(self, ctx, *, text):
...
+        responses = ["🍋" , "🍊", "🍉", ":seven:", ]
+        embed=discord.Embed(title="🎰 Slot Machine 🎰", description=random.choice(responses) + random.choice(responses) + random.choice(responses), color=0x176cd5)
+        embed.set_footer(text="You need triple 7's to win.")
+        await ctx.send(embed=embed)
+        
+    @cog_ext.cog_slash(name="slots", description="Play a simple game of slots")
+    async def slots(self, ctx):
+        responses = ["🍋" , "🍊", "🍉", ":seven:", ]
+        embed=discord.Embed(title="🎰 Slot Machine 🎰", description=random.choice(responses) + random.choice(responses) + random.choice(responses), color=0x176cd5)
+        embed.set_footer(text="You need triple 7's to win.")
```
Comment (thatjar) [inline cogs/fun.py:345]: You could add an `if` statement that checks if `embed.description` is three sevens and set the footer accordingly

**Review by thatjar — CHANGES_REQUESTED** [review]
I've left a comment, also the embed colour for 1Bot is `0xFF6600` (although it's not that important). Apart from those it seems good.

**Comment by ChaoticNebula5** [comment]
Alright, I'll try! Sorry for the late response, my internet died for a day.

**Comment by thatjar** [comment]
Oh it's fine, I already merged it :P

**Comment by ChaoticNebula5** [comment]
Oops just saw it xD

**Pull request merged**